servers/visual_server.cpp

```diff
@@ -2409,6 +2409,8 @@ VisualServer::VisualServer() {
 	GLOBAL_DEF_RST("rendering/quality/shadow_atlas/size", 4096);
 	GLOBAL_DEF("rendering/quality/shadow_atlas/size.mobile", 2048);
 	ProjectSettings::get_singleton()->set_custom_property_info("rendering/quality/shadow_atlas/size", PropertyInfo(Variant::INT, "rendering/quality/shadow_atlas/size", PROPERTY_HINT_RANGE, "256,16384"));
+	GLOBAL_DEF_RST("rendering/quality/shadow_atlas/cubemap_size", 512);
```

Comment:
Could you also sync classref and add the documentation for this setting?

Reply:
Sure thing! Thanks for the reminder.